Tests/SentryTests/SentryClientTests.swift

```diff
@@ -58,8 +58,22 @@ class SentryClientTest: XCTestCase {
         super.setUp()
         fixture = Fixture()
     }
-
+    
     func testCaptureMessage() {
+        let eventId = fixture.getSut().capture(message: message, scope: nil)
+
+        XCTAssertNotNil(eventId)
+
+        fixture.assertLastSentEvent { actual in
```

Comment:
M: I feel that `fixture.assertLastSentEvent` is being shared across tests, how do the ios runners work? do they create an instance of the Fixture on each test? or only because you `getSut()`? ideally, you don't assert the state of the inners fields of Fixture, you only tell them what they should return, and you assert on the result of your `getSut().your_actions`

Reply:
I init a new fixture on `setUp` for each test so they are not shared. The problem is that I need to access `transport` in order to find out if the event was passed properly to it. Doing this without a state is hard to achieve. Still, we could make a few methods static.